connection_manager.go

```diff
@@ -313,7 +332,7 @@ func (n *connectionManager) makeTrafficDecision(localIndex uint32, now time.Time
 		return closeTunnel, hostinfo, nil
 	}
 
-	primary := n.hostMap.Hosts[hostinfo.vpnIp]
+	primary := n.hostMap.Hosts[hostinfo.vpnAddrs[0]]
```

Comment:
I think we will want to find the matching network here instead of using the first vpn addr like in https://github.com/slackhq/nebula/pull/1261